xml/System.Threading/LazyInitializer.xml

```diff
@@ -114,7 +114,7 @@ ExpensiveData dataToUse = LazyInitializer.EnsureInitialized(ref _data, ref _data
       </Parameters>
       <Docs>
         <typeparam name="T">The type of the reference to be initialized.</typeparam>
-        <param name="target">A reference to initialize if it has not already been initialized.</param>
+        <param name="target">A reference to initialize if it has not already been initialized. If it is <see langword="null" />, it is considered not initialized; otherwise, it's considered not initialized.</param>
```

Comment:
something seems wrong here. Both sentences say it's considered not initialized.

should it be:

```suggestion
        <param name="target">A reference to initialize if it has not already been initialized. If it is <see langword="null" />, it is considered not initialized; otherwise, it's considered initialized.</param>
```